### PR TITLE
Fix app management not providing updates for apps which got disabled

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: lib/private/Installer.php [hunk 1]
```diff
@@ -389,7 +389,7 @@ public function isUpdateAvailable($appId) {
 			if($app['id'] === $appId) {
 				$currentVersion = OC_App::getAppVersion($appId);
 				$newestVersion = $app['releases'][0]['version'];
-				if (version_compare($newestVersion, $currentVersion, '>')) {
+				if ($currentVersion !== '0' && version_compare($newestVersion, $currentVersion, '>')) {
```
Comment (rullzer) [inline lib/private/Installer.php]: when will it be '0'?

Comment (rullzer) [reply lib/private/Installer.php]: Ah I should read the commit messages

**Review by rullzer — APPROVED** [review]
All makes sense!

File: lib/private/Installer.php (hunk 1 above)
Comment (nickvergessen) [reply lib/private/Installer.php]: https://github.com/nextcloud/server/blob/f30c14a6a27a937df8f53c056293ff15c0e3755c/lib/private/App/AppManager.php#L394-L397
When the app is not "available". so basically when it's only in the response of the appstore but can not be found locally

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**